queries.sql

```diff
@@ -15,3 +15,107 @@ SELECT * FROM animals WHERE neutered IS TRUE;
 SELECT * FROM animals WHERE name != 'Gabumon';
 
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+/* update the animals table by setting the species column to unspecified */
+--- Begin Transaction
+BEGIN;
+
+--- Update species col to 'unspecified'
+UPDATE animals
+SET species = 'unspecified';
+
+--- Verify that change was made
+SELECT * from animals;
+
+--- Rollback changes
+ROLLBACK;
+
+--- Verify changes
+SELECT * from animals;
+
+
+/* Update species col based on a condition */
+--- Begin Transaction
+BEGIN;
+
+--- update species col for animals with names ending in mon
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+--- update species col for animals with names not ending in mon
+UPDATE animals
+SET species = 'pokemon'
+WHERE name NOT LIKE '%mon';
+
+--- commit transaction
+COMMIT;
+
+--- verify changes
+SELECT * from animals;
+
+
+/* Delete all records in the animals table */
+BEGIN;
+
+--- delete all records
+DELETE from animals;
+
+--- verify all records are deleted
+SELECT * from animals;
+
+--- Rollback changes
+ROLLBACK;
+
+--- verify all records still exist
+SELECT * from animals;
+
+/* Queries */
+--- Begin transaction
+BEGIN;
+
+--- Delete all animals born after Jan 1st, 2022
+DELETE from animals 
+WHERE date_of_birth > '01-01-2022';
+
+--- create a savepoint
+SAVEPOINT SP1;
+
+--- update all animals' weights multiplied by -1
+UPDATE animals
+SET weight_kg = weight_kg * -1;
+
+--- roolback to savepoint
+ROLLBACK TO SP1;
+
+--- update all animals' weights that are negative multiplied by -1
+UPDATE animals
+SET weight_kg = weight_kg * -1
+WHERE weight_kg < 0;
+
+--- commit the transaction
+COMMIT;
+
+
```

Comment:
- [ ] Kindly add these data manipulation queries to data.sql file because these queries are manipulating data but queries.sql file only responsible for having data retieval queries 😊

Reply:
The tasks required that we add the transactions to the `queries,sql`. Please see below the screenshot.

![image](https://user-images.githubusercontent.com/20049193/205731504-7ba6cb39-5005-45a9-b123-19f2309733df.png)